File: source/repos/BlackJack/BlackJack.ViewModels/HistoryViews/GetAllGamesHistoryView.cs
using BlackJack.DataAccess.Enums;
using System;
using System.Collections.Generic;

namespace BlackJack.ViewModels.HistoryViews
{
    public class GetAllGamesHistoryView
    {
        public List<GameGetAllGamesHistoryViewItem> Games { get; set; }

        public GetAllGamesHistoryView()
        {
            Games = new List<GameGetAllGamesHistoryViewItem>();
        }
    }
    public class GameGetAllGamesHistoryViewItem
    {
        public Guid Id { get; set; }
        public int NumberOfBots { get; set; }
        public StatusType Status { get; set; }
        public string Winner { get; set; }
    }
}

File: source/repos/BlackJack/BlackJack.WEB/ClientApp/src/app/shared/entities/history/get-all-games-history.view.ts
import { StatusType } from 'src/app/shared/enums/status-type.enum.view';

export class GetAllGamesHistoryView {
    totalGamesCount:number;
    games:GameGetAllGamesHistoryViewItem[] = [];
}
export class GameGetAllGamesHistoryViewItem {
    id:string;
    numberOfBots:number;
    status:StatusType;
    winner:string;
}